lms/templates/certificates/_accomplishment-rendering.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
Some of the strings in this file use the `{| h}` filter. Will that conflict with this (and result in double escaping)?